components/EmojiPicker.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <button class="btn btn-outline-primary me-2 mt-2 "
+   @click="toggle">
+   <i class="bi bi-emoji-smile"></i>
+   EMO</button>
+   <Picker
+    v-if="emojiPickerSelected"
+    :data="emojiIndex"
+    set="apple" 
+    title="Pick your emoji…"
+    emoji="point_up"
+    @select="convertEmoji"
+    class="emoji-mart-category"
+    :style="{ backgroundColor: '#222529', color: '#FFFFFF',  }" 
```

Comment:
@Nidz-Web3 Could this style be added to the `emoji-mart-category` class in the CSS file instead? I prefer to have styles in one place (in .css files) instead of the inline CSS :)